### PR TITLE
Com

### PR DESCRIPTION
**Pull request by littlebeer2100**
add istio and kubesphere components scanner

### CODE REVIEW

File: pkg/models/components.go [hunk 1]
```diff
@@ -228,9 +230,9 @@ func GetComponents() (result []Components, err error) {
 
 	}
 
-	templates = []string{"kube-dns", "heapster", "monitoring-influxdb", "iam", "openpitrix"}
+	templates = []string{"kube-dns", "heapster", "monitoring-influxdb", "iam", "openpitrix","istio","kubesphere"}
 
-	namespaces := []string{KUBESYSTEM, OPENPITRIX}
+	namespaces := []string{KUBESYSTEM, OPENPITRIX,ISTIO,KUBESPHERE}
```
Comment (zryfish) [inline pkg/models/components.go]: define as a constant variable in constants.go would be better

File: pkg/models/components.go [hunk 2]
```diff
@@ -228,9 +230,9 @@ func GetComponents() (result []Components, err error) {
 
 	}
 
-	templates = []string{"kube-dns", "heapster", "monitoring-influxdb", "iam", "openpitrix"}
+	templates = []string{"kube-dns", "heapster", "monitoring-influxdb", "iam", "openpitrix","istio","kubesphere"}
```
Comment (zryfish) [inline pkg/models/components.go]: same here

Comment (zryfish) [reply pkg/models/components.go]: Also need to format code before check in

File: pkg/models/components.go (hunk 1 above)
Comment (littlebeer2100) [reply pkg/models/components.go]: constant will be handle together,for now there are in self model

**Pull request closed without merging**